### DISCUSSION TITLE
Can't change ID's

### DISCUSSION THREAD

**Discussion by Telejean** [desagas/Biome-Id-Fixer#6] [Q&A]
I updated my minecraft modpack to 1.16.5 from 1.15.2 and BOP biome id's got scrabled. If I interchange the 141:bamboo_blossom_grove with 59:alps I will still be in bamboo_blossom_grove and If i delete bamboo_blossom_grove altogether from BiomeIdFixer it will just create a new bamboo_blossom_grove with a new id.

**Comment by Telejean** [comment]
> I updated my minecraft modpack to 1.16.5 from 1.15.2 and BOP biome id's got scrabled. If I interchange the 141:bamboo_blossom_grove with 59:alps I will still be in bamboo_blossom_grove and If i delete bamboo_blossom_grove altogether from BiomeIdFixer it will just create a new bamboo_blossom_grove with a new id.

Turns out Chocolate was the culprint, so don't use Chocolate I guess.

**Reply by desagas** [reply]
Sometimes text editors apply styles to txt files, which is strange, and sometimes the file won't respond to changes, and I do not know why, except that it is related to ascii and metadata in the file created by some editors; and, if the pack is already loaded, rarely this happens as well, but, deleting the old file, and letting it create a new one, and then copying and pasting back over from the old file sometimes helps. Something to do with new lines, or returns, IDK. As for generating a new ID when you delete the old one, this is intended.
